plugin.yml

```diff
@@ -4,13 +4,19 @@ description: "Apply code signatures and sign MacOS binaries"
 author: https://github.com/DoomGerbil
 requirements:
   - security
```

Comment:
optional nit: maybe alpha-sort these?

Reply:
I split them into "apple tools" and "third party", and then sorted those - avoid people researching how to install obscure apple items.